fastlane/Fastfile

```diff
@@ -105,4 +149,96 @@ platform :android do
+    upload_to_play_store(
+      package_name:               'org.catrobat.catroid.phiro',
+      track:                      'alpha',
+      json_key_data:              ENV['googlePlayJsonKey'],
+      metadata_path:              './fastlane/metadata/android',
+      skip_upload_apk:            'true',
+      skip_upload_metadata:       'false',
+      skip_upload_images:         'false',
+      skip_upload_screenshots:    'false',
+      validate_only:              'false',
+      check_superseded_tracks:    'false',
+    )
+  end
+
+  desc "upload Phiro APK to alpha track"
+  lane :upload_APK_Phiro do
+    upload_to_play_store(
+      package_name:               'org.catrobat.catroid.phiro',
+      track:                      'alpha',
+      json_key_data:              ENV['googlePlayJsonKey'],
+      apk_paths:                  './catroid/build/outputs/apk/phiro/signedRelease/catroid-phiro-signedRelease.apk',
+      skip_upload_apk:            'false',
+      skip_upload_metadata:       'true',
+      skip_upload_images:         'true',
+      skip_upload_screenshots:    'true',
+    )
+  end
+
+  desc "promote Phiro APK on alpha track to production"
+  lane :promote_Phiro do
+    upload_to_play_store(
+      package_name:               'org.catrobat.catroid.phiro',
+      track:                      'alpha',
+      track_promote_to:           'production',
+      json_key_data:              ENV['googlePlayJsonKey'],
+      metadata_path:              './fastlane/metadata/android',
+      skip_upload_apk:            'true',
+      skip_upload_metadata:       'true',
+      skip_upload_images:         'true',
+      skip_upload_screenshots:    'true',
+    )
+  end
+
+  desc "upload Playground metadata only"
+  lane :upload_Metadata_Playground do
+    upload_to_play_store(
+      package_name:               'org.catrobat.catroid.test',
+      track:                      'alpha',
+      json_key_data:              ENV['googlePlayJsonKey'],
+      metadata_path:              './fastlane/metadata/android',
+      skip_upload_apk:            'true',
+      skip_upload_metadata:       'false',
+      skip_upload_images:         'false',
+      skip_upload_screenshots:    'false',
+      validate_only:              'false',
+      check_superseded_tracks:    'false',
+    )
+  end
+
+  desc "upload Playground APK to alpha track"
+  lane :upload_APK_Playground do
+    upload_to_play_store(
+      package_name:               'org.catrobat.catroid.test',
+      track:                      'alpha',
+      json_key_data:              ENV['googlePlayJsonKey'],
+      apk_paths:                  './catroid/build/outputs/apk/catroid/signedRelease/catroid-catroid-signedRelease.apk',
+      skip_upload_apk:            'false',
+      skip_upload_metadata:       'true',
+      skip_upload_images:         'true',
+      skip_upload_screenshots:    'true',
+      validate_only:              'false',
+      check_superseded_tracks:    'false',
+    )
```

Comment:
guess there is no Promotion for Playground APK right? - so this step can't be tested with this...

Reply:
Exactly! The Playground APK is not supposed to be published to the public (production track) anyway. It is only published to the alpha track.